uvtools/plot.py

```diff
@@ -466,3 +469,265 @@ def omni_view_gif(filenames, name='omni_movie.gif'):
+    axes = fig.subplots(len(plot_type), 3)
+    axes = [axes,] if len(axes.shape) == 1 else axes # avoid bug for single row
+    axes[0][0].set_title("Amplitude Difference", fontsize=12)
+    axes[0][1].set_title("Phase Difference", fontsize=12)
+    axes[0][2].set_title("Amplitude of Complex Difference", fontsize=12)
+
+    # helper function for getting the extent of axes
+    extent = lambda xvals, yvals : (xvals[0], xvals[-1], yvals[-1], yvals[0])
+
+    # loop over items in visibilities and plot them
+    for i, item in enumerate(visibilities.items()):
+        # extract visibilities, get diffs
+        visA, visB = item[1].values()
+        diffs = (utils.diff(visA, visB, 'abs'), 
+                 utils.diff(visA, visB, 'phs'),
+                 utils.diff(visA, visB, 'complex'))
+
+        # extract parameters
+        ykey, xkey = item[0].split("_vs_") # keys for choosing parameters
+        xvals, yvals = plot_params[xkey], plot_params[ykey]
+
+        # get labels
+        xlabel, ylabel = labels[xkey], labels[ykey]
+
+        # plot stuff
+        for ax, diff in zip(axes[i], diffs):
+            # set labels
+            ax.set_xlabel(xlabel, fontsize=12)
+            ax.set_ylabel(ylabel, fontsize=12)
+
+            # plot waterfall and add a colorbar
+            fig.sca(ax)
+            cax = waterfall(diff, mode="real", cmap='viridis', 
+                            extent=extent(xvals, yvals))
+            fig.colorbar(cax)
+
+    # display the figure
+    plt.tight_layout()
+    plt.show()
+
+    # save if desired
+    if save_path is not None:
+        utils.savefig(fig, save_path)
+
+def plot_diff_uv(uvd1, uvd2, pol=None, speedup=True,
+                 skip_check=False, save_path=None,
+                 resolution=50):
+    """Summary plot for difference between visibilities.
+
+    Parameters
+    ----------
+    uvd1, uvd2 : pyuvdata.UVData
+        Input UVData objects which contain the visibilities to be differenced 
+        and any relevant metadata.
+
+    pol : str, None, optional
+        String specifying which polarization to be used. Must be one of the
+        polarizations listed in the UVData.get_pols() method for both
+        `uvd1` and `uvd2`. Default is to use the 0th polarization.
+
+    speedup : bool, optional
+        Whether to use the fast implementation of this plotting tool or the
+        slower implementation. The faster implementation does not produce
+        figures as nice as the slower implementation, but the slower
+        implementation is only really usable for small sets of data.
+        Default is to use the fast implementation.
+
+    skip_check : bool, optional
+        Whether to check that the metadata for `uvd1` and `uvd2` match.
+        Default behavior is to check the metadata.
+
+    save_path : str, optional
+        Path to where the figure should be saved; may be absolute or relative.
+        Default is to not save the figure.
+    
+    resolution : int, optional
```

Comment:
I would call this `bins` instead of `resolution`, to match better with the usage of the matplotlib histogram functions.